### PR TITLE
ACTION REQUIRED for MountPropagation feature

### PR DESCRIPTION
**Pull request by andrewsykim**
Update CHANGELOG re: https://github.com/kubernetes/kubernetes/issues/61058. 

Wasn't sure if these docs are autogenerated. 

```release-note
ACTION REQUIRED: MountPropagation feature is now beta. As a result, all volume mounts in containers are now "rslave" on Linux by default. Any host paths that is not a `rslave` mount or a `rshared` mount will not work. On Linux, you can ensure that mounts are of type `rshared` by running `mount --make-rshared /path/to/mount`, otherwise set `MountPropagation=false` on the kubelet `--feature-gates` flag. ([#59252](https://github.com/kubernetes/kubernetes/pull/59252), [@jsafrane](https://
```

### CODE REVIEW

File: CHANGELOG-1.10.md [hunk 1]
```diff
@@ -176,6 +176,7 @@ filename | sha256 hash
...
 * Action Required: The boostrapped RBAC role and rolebinding for the `cloud-provider` service account is now deprecated. If you're currently using this service account, you must create and apply your own RBAC policy for new clusters. ([#59949](https://github.com/kubernetes/kubernetes/pull/59949), [@nicksardo](https://github.com/nicksardo))
 * ACTION REQUIRED: VolumeScheduling and LocalPersistentVolume features are beta and enabled by default.  The PersistentVolume NodeAffinity alpha annotation is deprecated and will be removed in a future release. ([#59391](https://github.com/kubernetes/kubernetes/pull/59391), [@msau42](https://github.com/msau42))
+* ACTION REQUIRED: MountPropagation feature is now beta. As a result, all volume mounts in containers are now "rslave" on Linux by default. Any host paths that is not a `rslave` mount or a `rshared` mount will not work. On Linux, you can ensure that mounts are of type `rshared` by running `mount --make-rshared /path/to/mount`, otherwise set `MountPropagation=false` on the kubelet `--feature-gates` flag. ([#59252](https://github.com/kubernetes/kubernetes/pull/59252), [@jsafrane](https://github.com/jsafrane))
```
Comment (gnufied) [inline CHANGELOG-1.10.md]: I do not think this is accurate. This should read - "MountPropagation feature is now beta. As a result, all volume mounts in containers are now "rslave" on Linux by default. To make this default work in all Linux environments-  you should have entire mount tree marked as shareable via `mount --make-rshared /` .  All Linux distributions that use systemd already have root directory mounted as `rshared` and hence they need not do anything.  In Linux environments without systemd we also recommend restarting docker daemon after marking root directory as `rshared`"

Comment (andrewsykim) [reply CHANGELOG-1.10.md]: Will update, thanks :). It seems this is a required feature for CSI, is that worth mentioning here too?

**Comment by cblecker** [comment]
cc @nickchase @jdumars 

I'm not sure this is the right place for this update. CC'ing the release notes lead

File: CHANGELOG-1.10.md (hunk 1 above)
Comment (gnufied) [reply CHANGELOG-1.10.md]: We can leave it as it is. The CSI docs will cover this requirement in detail.

**Comment by jdumars** [comment]
Good catch.  This should not be merged. The details will go into the release notes draft.

**Comment by andrewsykim** [comment]
I left a referene to this PR in the release notes draft.

**Pull request closed without merging**